addon/start-mirage.js

```diff
@@ -0,0 +1,52 @@
+import Ember from 'ember';
+import readModules from 'ember-cli-mirage/utils/read-modules';
+import Server from 'ember-cli-mirage/server';
+import _assign from 'lodash/assign';
+
+const {
+  getWithDefault
+} = Ember;
```

Comment:
Can we use `import { getWithDefault } from '@ember/object';`?

Reply:
Sure...I guess I left it this way because it's what the modules API codemod produces. Maybe a bug with the codemod?

Reply:
Ya, possibly. Might need to make sure it has the newest mapping’s....